lib/PlaidClient.js

```diff
@@ -210,6 +210,42 @@ Client.prototype.getTransactions =
     }, options, cb);
   };
 
+// getAllTransactions(String, Date, Date, Object?, Function)
+Client.prototype.getAllTransactions =
+  async function(access_token, start_date, end_date, cb) {
```

Comment:
@evanlimanto, lets write this without using mode modern node features. I can't imagine that all of our clients have node on `7.x` right now.

I would consider looking at http://bluebirdjs.com/docs/api/promise.coroutine.html